openstack/nova/values.yaml

```diff
@@ -744,6 +744,27 @@ mysql_metrics:
         UNION ALL SELECT 'i-uuid', 'source', 'dest', 0;
       values:
         - "gauge"
+    - help: Multiple BDM Volume Attachments
+      name: openstack_compute_multiple_bdm_attachments_gauge
+      labels:
+        - "instance_uuid"
+        - "volume_id"
+      query: |
+        SELECT count(*) AS gauge, bdm.instance_uuid, bdm.volume_id
+        FROM
+          block_device_mapping AS bdm
+          JOIN instances AS i ON i.uuid=bdm.instance_uuid
+        WHERE
+          i.deleted=0
+          AND bdm.deleted=0
+          AND bdm.volume_id IS NOT NULL
+          AND bdm.attachment_id IS NOT NULL
```

Comment:
Why is this relevant? Wouldn't they be still bad? They could at least block a customer from detaching the volume if the connection_info is empty and the empty is was created later and thus is chosen by Nova.